src/pages/Calculator.tsx

```diff
@@ -186,60 +179,63 @@ export const Calculator = (): React.ReactElement => {
           </Card>
         </Col>
 
-        <Col md="12" lg="8">
+        <Col md="12" lg="8" id="activity-section">
           <Card id="person-risk">
             {prevalenceIsFilled ? (
               <React.Fragment>
                 <header id="activity-risk">
                   Step 2: Describe the activity
                 </header>
                 <div>
-                  <SavedDataSelector
-                    currentData={calculatorData}
-                    setter={setCalculatorData}
-                  />
-
-                  <SelectControl
+                  <GenericSelectControl
                     id="interaction"
                     label="Is this a single activity or an ongoing relationship?"
-                    data={calculatorData}
-                    setter={setCalculatorData}
+                    setter={(value) =>
+                      setCalculatorData({
```

Comment:
Add a comment here for why this is special.

Reply:
Done